source/extensions/filters/network/dubbo_proxy/stats.h

```diff
@@ -33,7 +33,7 @@ namespace DubboProxy {
   COUNTER(response_error_caused_connection_close)                                                  \
   COUNTER(response_success)                                                                        \
   GAUGE(request_active, Accumulate)                                                                \
-  HISTOGRAM(request_time_ms)
+  HISTOGRAM(request_time, Milliseconds)
```

Comment:
again we need to be careful to message out how we avoid breaking monitoring for people.

Reply:
<Consolidated the rename discussion in the OP>